### PR TITLE
Secret

### PR DESCRIPTION
**Pull request by david30907d**
**What does your PR do?**

1. add a yaml template for developer to create `EKS` cluster on Amazon
2. fine-tune some performance settings

## Checklist

- [x] Commits are [signed](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#sign-your-work)
- [ ] Commits are [squashed](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#squash-commits) (if appropriate)
- [x] Chart [version bumped](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#versioning)
- [x] Chart [documentation updated](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#documentation)
- [x] Passes [linting](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#linting)

### CODE REVIEW

**Comment by david30907d** [comment]
c.c. @tresl-kc

**Review by tresl-kc — APPROVED** [review]
👀

**Comment by david30907d** [comment]
forgot to explain what i've done in the previous PR:
1. need to specify Image in Chart (doesn't put our code in the container so put it to my repo preventing to setup docker secrets 😅 ) : <https://github.com/tresl-co/charts/blob/1a20e2f556778bbae02c057d82b3708dceb15d18/charts/airflow/examples/minikube/custom-values.yaml#L16-L17>
2. Setup autoscaling policy here: <https://github.com/tresl-co/charts/blob/1a20e2f556778bbae02c057d82b3708dceb15d18/charts/airflow/examples/minikube/custom-values.yaml#L298-L307>
3. Dag would sync every 60s: <https://github.com/tresl-co/charts/blob/1a20e2f556778bbae02c057d82b3708dceb15d18/charts/airflow/examples/minikube/custom-values.yaml#L325-L383>
4. Would apply CI/CD on master/prod branch: <https://github.com/tresl-co/charts/blob/1a20e2f556778bbae02c057d82b3708dceb15d18/charts/airflow/examples/minikube/custom-values.yaml#L336>

c.c. @tresl-kc

File: charts/airflow/eksctl-create-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# An example of ClusterConfig object using an existing VPC:
```
Comment (david30907d) [inline charts/airflow/eksctl-create-cluster.yaml:1]: Use this yaml to create cluster on `AWS eks`

FYI: <https://www.notion.so/tresl/ETL-f20df50e57af459ea5b0cdacc874d95e#7750063f809b484a8fd63cd708b24902>

**Review by tonyin — APPROVED** [review]
show us how to use this later

**Comment by david30907d** [comment]
> show us how to use this later

roger that, will write up a notion

**Pull request merged**